example/12_reduce/CMakeLists.txt

```diff
@@ -1 +1,2 @@
-add_example_executable(example_reduce_blockwise reduce_blockwise.cpp -D 16,64,32,960 -v 1 1 10)
+add_example_executable(example_reduce_blockwise reduce_blockwise.cpp)
```

Comment:
CI shows you are missing program args for testing example code for both `example_reduce_blockwise` and `example_reduce_blockwise_two_call`

Reply:
Fixed.   Use default parameters when the executable is called by CI

Reply:
Allowing passing cmd arguments this way was from #233